permission_handler/example/lib/main.dart

```diff
@@ -1,127 +1,154 @@
+            height: 1.2,
+          ),
+          button: TextStyle(color: Colors.white),
+          headline1: TextStyle(
+            color: Colors.white,
+            fontSize: 18,
+          ),
         ),
-        body: Center(
-          child: ListView(
-              children: Permission.values
-                  .where((Permission permission) {
-                    if (Platform.isIOS) {
-                      return permission != Permission.unknown &&
-                          permission != Permission.sms &&
-                          //permission != Permission.storage &&
-                          permission != Permission.ignoreBatteryOptimizations &&
-                          permission != Permission.accessMediaLocation;
-                    } else {
-                      return permission != Permission.unknown &&
-                          permission != Permission.mediaLibrary &&
-                          permission != Permission.photos &&
-                          permission != Permission.reminders;
-                    }
-                  })
-                  .map((permission) => PermissionWidget(permission))
-                  .toList()),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        inputDecorationTheme: InputDecorationTheme(
+          fillColor: const Color.fromRGBO(37, 37, 37, 1),
+          filled: true,
         ),
       ),
+      home: AppHome(title: 'Baseflow $pluginName example app'),
     );
   }
-}
 
-/// Permission widget which displays a permission and allows users to request
-/// the permissions.
-class PermissionWidget extends StatefulWidget {
-  /// Constructs a [PermissionWidget] for the supplied [Permission].
-  const PermissionWidget(this._permission);
+  /// Creates [MaterialColor] based on [Color]
+  static MaterialColor createMaterialColor(Color color) {
+    List strengths = <double>[.05];
+    Map swatch = <int, Color>{};
+    final r = color.red, g = color.green, b = color.blue;
 
-  final Permission _permission;
-
-  @override
-  _PermissionState createState() => _PermissionState(_permission);
+    for (var i = 1; i < 10; i++) {
+      strengths.add(0.1 * i);
+    }
+    for (var strength in strengths) {
+      final ds = 0.5 - strength;
+      swatch[(strength * 1000).round()] = Color.fromRGBO(
+        r + ((ds < 0 ? r : (255 - r)) * ds).round(),
+        g + ((ds < 0 ? g : (255 - g)) * ds).round(),
+        b + ((ds < 0 ? b : (255 - b)) * ds).round(),
+        1,
+      );
+    }
+    return MaterialColor(color.value, swatch);
+  }
 }
 
-class _PermissionState extends State<PermissionWidget> {
-  _PermissionState(this._permission);
+/// A Flutter example demonstrating how the [pluginName] plugin could be used
+class AppHome extends StatefulWidget {
+  /// Constructs the [AppHome] class
+  AppHome({Key key, this.title}) : super(key: key);
 
-  final Permission _permission;
-  PermissionStatus _permissionStatus = PermissionStatus.undetermined;
+  /// Application title
```

Comment:
Adhere to the Effective Dart style guidelines regarding to Doc comments, specifically rule "PREFER starting variable, getter, or setter comments with noun phrases." (https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases)

```suggestion
  /// The [title] of the application, which is shown in the application's title bar.
```